cognite/extractorutils/uploader/time_series.py

```diff
@@ -79,9 +84,15 @@ def default_time_series_factory(external_id: str, datapoints: DataPointList) ->
     return TimeSeries(external_id=external_id, is_string=is_string)
 
 
-class TimeSeriesUploadQueue(AbstractUploadQueue):
+def default_cdm_time_series_factory(
+    create_these_ids: set[NodeId], timeseries_apply_dict: dict[NodeId, CogniteExtractorTimeSeriesApply]
+) -> list[CogniteExtractorTimeSeriesApply]:
+    return [timeseries_apply_dict[nodeId] for nodeId in create_these_ids]
```

Comment:
This still works quite differently from the legacy TS queue.

The legacy queue takes an id and a list of datapoints in the `add_to_upload_queue` method, and calls the callback whenever it needs to go from id to full object (ie, when creating)

The new queue takes the full node object every call to add, in addition to the datapoints. And this callback does nothing.

The `add_to_upload_queue` method should take in a `NodeId` and a set of datapoints, and the callback should be called whenever the `CogniteExtractorTimeSeriesApply` actually needs to be generated.